a10_neutron_lbaas/etc/defaults.py

```diff
@@ -37,6 +37,7 @@
 DEVICE_OPTIONAL_DEFAULTS = {
     "protocol": "https",
     "port": 443,
+    "conn-limit": 8000000,
```

Comment:
We settled on "if unspecified, don't put it in the request" - this is the best of both worlds. Gonna close this request to kick the test infrastructure.